BraveRewardsUI/Ads/AdContentButton.swift

```diff
@@ -97,4 +110,16 @@ class AdContentButton: UIControl {
       }
     }
   }
+  
+  func setupTheme() {
```

Comment:
Can we change this function name to `applyTheme`? Just being a little picky because the rest of the app uses that name that's why.

Other than that, approved for me :)

Reply:
I sure can :)